cns-hook

```diff
@@ -73,6 +75,12 @@ function deploy_challenge {
 		verifyvm "$domain" "$vmuuid"
 		update_remotevm "$vmuuid" "$tokenval"
 		;;
+	sdcsso)
+		local admin=$(sdc-useradm get admin | json uuid)
+		vmuuid=$(sdc-vmadm list -H -o uuid owner_uuid="$admin" alias="sdcsso" | tail -1)
+		verifyvm "$domain" "$vmuuid"
+		update_remotevm "$vmuuid" "$tokenval"
```

Comment:
You've used `update_remotevm` here, but further down you're assuming that the `sdcsso` VM is on the headnode (by using `zlogin`, direct `cp`, etc.)

Do `sdcsso` zones normally get deployed directly on the headnode? If so, and we're ok with assuming it here, this should be an `update_localvm` call instead.

Reply:
It's not always on the headnode. In two of our three deployments it's not on the headnode.